### ISSUE TITLE
Paragraph and line breaks not working in Quote posts on the dash

### ISSUE THREAD

**Issue by kaithaba** [new-xkit/XKit#883]
I've got a problem where if I make a quote post, especially if it's a poem or something where I've divided the text down to several short lines, the paragraph breaks won't show up on the dashboard. The first verse of a poem might have a working paragraph break after it, but then all the other verses that come after it will be just one long column of text. (The post will look fine when I'm on my blog page, however--this only happens on the dashboard, and when I go in to edit the post, it has put in the HTML for </br> and </p>.) I've had a similar problem with old text posts recently as well and it's even worse: even the line breaks disappear, and they get mushed into just one big wall of text, and that happens on the blog pages themselves as well. I know Tumblr has always had stupid issues with paragraph breaks, but this seems to be a new one. Anyone else seen it?

**Comment by nightpool** [comment]
Can you link a post that's broken?

**Comment by kaithaba** [comment]
Here's what happened to a text post (this is visible on the blog page itself) http://aikainkauna.tumblr.com/post/130718986668/ficpoetry-my-dread-and-dire-suitor  And this is what it does to quote posts: http://kissinthedreamhouse.tumblr.com/post/115082264816/i-am-like-thee-o-night-dark-and-naked-i-walk That looks fine on the blog page (and the layout on that has really small line/paragraph breaks but they're there, I can see they're working), but when it's on the dashboard, it looks like this: 
![quotescrewed](https://cloud.githubusercontent.com/assets/16408765/11970898/fe3c0ada-a93b-11e5-89b6-c19d56430c87.jpg) The first paragraph break after the first verse is okay, but all the other ones after that are missing, even if they were typed in.

**Comment by nightpool** [comment]
Looks like this isn't an XKit problem—try reporting it to tumblr support
and see what they say?

**Comment by kaithaba** [comment]
Ok. Shall ask them what's up. And if they can't fix it and if others have the same problem, we can ask you to help and write something that fixes it, right? I'm sure it isn't just me...

**Comment by nightpool** [comment]
Yeah this is definitely an issue. should just be a one line change for them.

**Comment by kaithaba** [comment]
I contacted Tumblr support over this and they just asked me to change my layout, but that's not helpful. Can you give me something I could forward to them about this 'one line change' issue so they'd know what to do with it? Thanks in advance.

**Comment by nightpool** [comment]
As I said in #1058, I'm following up with tumblr on this one. 

Here is the support ticket I filed: https://gist.github.com/nightpool/dd5976f4dd7b7221ba725fab77c8e491

**Comment by kaithaba** [comment]
It's almost a year and Tumblr still haven't done anything about it:/ Could you code in a tweak into Xkit that fixes it, since Tumblr doesn't seem to care?
